usb/device/targets/TARGET_RENESAS/TARGET_RZ_A1XX/USBEndpoints_RZ_A1.h

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2010-2011 mbed.org, MIT License
```

Comment:
@TomoYamanaka I'd br curious to hear about the history of this file since this header makes me think it's beein around for a while.

Reply:
Sorry, this is new file. It seems that the information of the reference file has remained. I modified this file's header.